types/bool.go

```diff
@@ -79,14 +79,19 @@ func (b Bool) Equal(other attr.Value) bool {
 	return b.Value == o.Value
 }
 
+// IsNull returns true if the Bool represents null value.
 func (b Bool) IsNull() bool {
 	return b.Null
 }
 
+// IsUnknown returns true if the Bool represents currently unknown value.
```

Comment:
Nit: Missing article here and other IsUnknown

```suggestion
// IsUnknown returns true if the Bool represents a currently unknown value.
```